cmd/tendermint/commands/debug/kill.go

```diff
@@ -33,10 +33,16 @@ $ tendermint debug kill 34255 /path/to/tm-debug.zip`,
 		Args: cobra.ExactArgs(2),
 		RunE: func(cmd *cobra.Command, args []string) error {
 			ctx := cmd.Context()
-			pid, err := strconv.ParseInt(args[0], 10, 64)
+			// Using Atoi so that the size of an integer can be automatically inferred.
+			pid, err := strconv.Atoi(args[0])
 			if err != nil {
 				return err
 			}
+			if pid == 0 {
+				return errors.New("PID value must be > 0")
+			} else if pid < 0 {
+				return fmt.Errorf("PID value must be > 0; given value %q, got %d", args[0], pid)
+			}
```

Comment:
nit: I think just a single error in this case suffices

Reply:
I'd usually break them down as `"PID value must be > 0; given value "0", got "0"` is a non-informative message hence why the 2 cases, but I've made the update as requested.